testdata/p4_14_samples/switch_20160226/includes/intrinsic.p4

```diff
@@ -19,44 +19,46 @@ header_type ingress_intrinsic_metadata_t {
         resubmit_flag : 1;              // flag distinguishing original packets
                                         // from resubmitted packets.
 
-        ingress_global_tstamp : 48;     // global timestamp (ns) taken upon
+        ingress_global_timestamp : 48;  // global timestamp (ns) taken upon
                                         // arrival at ingress.
 
         mcast_grp : 16;                 // multicast group id (key for the
                                         // mcast replication table)
-
+#ifdef INCLUDE_OLD_INTRINSIC_METADATA_FIELDS
```

Comment:
Do we need this macro?

Reply:
If I remove that macro and the #ifdef and corresponding #endif, then three of the old intrinsic_metadata fields remain in the post-preprocessed program, and these changes are attempting to prevent that from happening, because simple_switch has no implementation of these fields.

I could delete that code entirely, but the reason I introduced a new macro was because I was thinking that perhaps someone somewhere might want to see the source code that accessed these old fields, and re-enable it.

If folks think that is not going to happen, I could simply delete the few dozen lines of code that are currently between such #ifdef / #endif pairs in the current proposed changes.